CHANGELOG.txt

```diff
@@ -17,6 +17,16 @@ Changes:
      (see #3068)
    * add support for models without a core or inner core
      (see #1200, #1423, #2086, #2192, #2390, #3070)
+ - obspy.io.nordic:
+   * add support for writing multiple amplitudes for one pick (only new format)
+     and writes some amplitude lines more compact
+   * add full support for reading and writing apparent velocity and backazimuth
+     with new nordic format.
+   * fix comments on angle of incidence vs. takeoff angle (What is called
+     angle of incidence in Seisan is the same as what is usually referred to as
+     takeoff angle)
+   * add function to check whether phase is valid IASPEI phase in Seisan
+   * full support for writing high-accuracy origin line
```

Comment:
can you add PR number to all the bullets please? =)

Reply:
fixed.